scipy/spatial/_spherical_voronoi.py

```diff
@@ -0,0 +1,360 @@
+"""
+Spherical Voronoi Code
+
+.. versionadded:: 0.17.0
+
+"""
+#
+# Copyright (C)  Tyler Reddy, Ross Hemsley, Edd Edmondson,
+#                Nikolai Nowaczyk, Joe Pitt-Francis, 2015.
+#
+# Distributed under the same BSD license as Scipy.
+#
+
+import numpy as np
+import numpy.matlib
+import scipy
+import itertools
+from scipy._lib._version import NumpyVersion
+
+# Whether Numpy has stacked matrix linear algebra
+HAS_NUMPY_VEC_DET = (NumpyVersion(np.__version__) >= '1.8.0')
+
+__all__ = ['SphericalVoronoi']
+
+
+def determinant_fallback(m):
+    """
+    Calculates the determinant of m using Laplace expansion in the first row.
+    This function is used only as a fallback to ensure backwards compatibility
+    with Python 2.6.
```

Comment:
If the issue is backwards compat with Numpy 1.8, then this function should not be needed, because there a numpy version check is made already before, cf. `if HAS_NUMPY_VEC_DET:` below.